lib/monitor/measure.go

```diff
@@ -107,6 +111,7 @@ func send(v interface{}) {
 	if !enabled {
 		return
 	}
+	// XXX comment needed:
```

Comment:
For a large number of clients (˜10'000), the connection phase can take some time. This is a linear backoff to enable connection even when there are a lot of requests.